### PR TITLE
CAL-787 Remove platform-util embeds and fix build

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR removes platform-util embeds and fixes the build.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@jlcsmith
@shaundmorris
@stustison

#### How should this be tested?
Build / Install / Verify basic functionality
#### What are the relevant tickets?
https://github.com/codice/alliance/issues/787
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

File: distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
         <bundle>mvn:ddf.thirdparty/restito/${ddf.version}</bundle>
 
-        <bundle>wrap:mvn:ddf.platform.util/platform-util/${ddf.version}</bundle>
+<!--        <bundle>wrap:mvn:ddf.platform.util/platform-util/${ddf.version}</bundle>-->
```
Comment (jlcsmith) [inline distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml]: ❓ remove?

File: catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java [hunk 2]
```diff
@@ -421,7 +420,7 @@ private void initCorba() throws InvalidName, AdapterInactive, WrongPolicy, Serva
   }
 
   public static synchronized Subject getGuestSubject() throws SecurityServiceException {
-    if (guestSubject == null || Security.getInstance().tokenAboutToExpire(guestSubject)) {
```
Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java]: ❓ why was this conditional removed?

Comment (bdeining) [reply catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java]: this method was removed from DDF

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by shaundmorris** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Codice/alliance/pullRequest?prid=4843817&bid=16079492)

**Comment by bdeining** [comment]
build now

**Pull request closed without merging**